mantidproject/mantid
# Update SANS solid angle correction and docs

After reviewing the SANS reduction documentation, we found that there was an error in the implementation of the SANS solid angle correction. This PR addresses the following:
1. Fix the solid angle correction in the case of tube detectors.
2. Update the SANSSolidAngleCorrection docs accordingly.
3. Update the ORNL SANS reduction docs accordingly.
4. While updating the docs, we also added a figure for the stitching command.

**To test:**
1. Make sure the system tests pass.
2. **Review the new implementation**. Compare the implementation against the new documentation.
3. Check that the updated docs read well and that the images and math show up.

There is no git issue for this PR. All the info is here.

Does not need to be in the release notes.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
